Service/src/main/java/SendingTheTelephone/TelephoneSending.java

```diff
@@ -0,0 +1,24 @@
+package SendingTheTelephone;
+
+import TIcket.Ticket;
+
+import java.time.format.DateTimeFormatter;
+
+public class TelephoneSending extends TicketSending {
```

Comment:
why have you chosen the extension here?)

Reply:
I have chosen extension because class TelephoneSending inherited from the class TicketSending. Classes TelephoneSending and EmailSending implement different ways of sending messages.

Reply:
I have rewrite this realization using interfaces. As for me it is more convenient.